src/ArborX_LinearBVH.hpp

```diff
@@ -232,15 +239,20 @@ BoundingVolumeHierarchy<MemorySpace, Enable>::BoundingVolumeHierarchy(
       "ArborX::BVH::BVH::calculate_scene_bounding_box");
 
   // determine the bounding box of the scene
+  Box bbox{};
   Details::TreeConstruction::calculateBoundingBoxOfTheScene(space, primitives,
-                                                            _bounds);
+                                                            bbox);
 
   Kokkos::Profiling::popRegion();
 
   if (size() == 1)
   {
     Details::TreeConstruction::initializeSingleLeafNode(
         space, primitives, _internal_and_leaf_nodes);
+    Kokkos::deep_copy(space,
+                      Kokkos::View<BoundingVolume, Kokkos::HostSpace>(&_bounds),
+                      Kokkos::View<BoundingVolume, MemorySpace>(
```

Comment:
I'm not exactly sure what's happening here. Why is it not an `UnmanagedView`? What is the View constructor that takes in a pointer?

Reply:
https://github.com/kokkos/kokkos/blob/6d65b5a3a1a48c43312a4f35f4eea367f59ec554/core/src/Kokkos_View.hpp#L1749-L1773
https://github.com/kokkos/kokkos/blob/6d65b5a3a1a48c43312a4f35f4eea367f59ec554/core/src/Kokkos_View.hpp#L1587-L1602
https://github.com/kokkos/kokkos/blob/6d65b5a3a1a48c43312a4f35f4eea367f59ec554/core/src/impl/Kokkos_ViewMapping.hpp#L3205-L3213

Reply:
I don't understand that code. However, I stumbled upon issue 278 in kokkos (don't want to link). Carter was saying
```
Will even work with "managed" view since constructing with a pointer makes it runtime-unmanaged, which is slightly greater overhead than trait-unmanaged.
```
While I still don't understand what `runtime-unmanaged` or `trait-unmanaged` mean, it still sounds like there is not reason to prefer this approach to just using unmanaged views.

Reply:
Less typing :). But I don't mind changing.

Reply:
Done